lib/rides/widgets/search_ride_filter.dart

```diff
@@ -35,8 +33,9 @@ class SearchSuggestionFilter {
   late int _minReliabilityRating;
   late int _minHospitalityRating;
   late List<Feature> _selectedFeatures;
-  late SearchSuggestionSorting _sorting;
-  final TextEditingController _maxDeviationController = TextEditingController();
+  late SearchRideSorting _sorting;
+
+  bool _wholeDay;
```

Comment:
I don't know about the whole day thing. So we are saying we filter for rides only at that day and you don't see anything the day after?